exporter/googlecloudexporter/factory.go

```diff
@@ -106,12 +105,10 @@ func createLogsExporter(ctx context.Context, set component.ExporterCreateSetting
 	processors := []component.LogsProcessor{}
 	processorConfigs := []config.Processor{
 		exporterConfig.BatchConfig,
-		exporterConfig.DetectConfig,
 	}
 
 	processorFactories := []component.ProcessorFactory{
 		batchprocessor.NewFactory(),
-		resourcedetectionprocessor.NewFactory(),
```

Comment:
@jmwilliams89 can you help explain to me why this is removed from metrics and logs but not traces?

Reply:
Awesome catch. That needs to be removed from all of them.